flintrock/flintrock.py

```diff
@@ -297,6 +305,21 @@ def launch(
         raise UnsupportedProviderError(provider)
 
 
+def get_last_commit_sha(git_repository: str):
+    """
+    Retrieve the last commit's SHA from a repository hosted on Github.
+    """
+    parsed_url = urllib.parse.urlparse(git_repository)
+    if parsed_url.netloc != "github.com":
+        raise UsageError('Error: Retrieving the last commit\'s SHA only works with '
```

Comment:
Oh, you're right. I did that because I was double-quoting option names to be consistent with Click. 

Hmm, never mind then. I've been inconsistent in this case; I'll fix it myself later.